docs/docsite/rst/porting_guides/porting_guide_6.rst

```diff
@@ -99,6 +99,30 @@ Networking
 
 No notable changes
 
+Porting Guide for v6.5.0
+========================
+
+Major Changes
+-------------
+
+infoblox.nios_modules
+~~~~~~~~~~~~~~~~~~~~~
+
+- Feature for extra layer security , with cert and key parameters in playbooks for authenticating using certificate and key *.pem file absolute path `#154 <https://github.com/infobloxopen/infoblox-ansible/pull/154>`_
```

Comment:
```suggestion
- Feature for extra layer security , with cert and key parameters in playbooks for authenticating using certificate and key \*.pem file absolute path `#154 <https://github.com/infobloxopen/infoblox-ansible/pull/154>`_
```

I believe this should resolve the issue with CI and the docs build.

Reply:
Thanks @sivel 
Also, jinx, I just pushed a similar fix:

```
$ cat test.rst 
- Feature for extra layer security, with cert and key parameters in playbooks for authenticating using certificate and key ``*.pem`` file absolute path

$ rstcheck test.rst 
```